libs/pyrt_component/wit/pyrt.wit

```diff
@@ -0,0 +1,56 @@
+package metatype:pyrt;
```

Comment:
Note, nothing in this wit if python specific. In another PR, will refactor this and the generation into `metagen`.